### PR TITLE
Add free tier callout for special scopes and alpha 400 error list

### PR DESCRIPTION
**Pull request by jcleblanc**
- Added callout that scopes available by request can only be enabled for paid tiers
- Moved metadata 400 error into alphabetical order

### CODE REVIEW

File: content/guides/api-calls/permissions-and-errors/common-errors.md [hunk 1]
```diff
@@ -113,6 +113,9 @@ for solution to common errors encountered when working with the Box APIs.
...
 |              |                                                                                                                                                                                                                                                                                                                                                                           |
+| **Error**    | `metadata_after_file_contents`                                                                                                                                                                                                                                                                                                                                            |
+| **Message**  | Metadata is included after file contents in a file upload request.                                                                                                                                                                                                                                                                                                        |
```
Comment (cbetta) [inline content/guides/api-calls/permissions-and-errors/common-errors.md:117]: This error seems weird. I didn't know you could apply Box Metadata at the same time as a file. Do we mean the `attributes` instead?

Comment (cbetta) [reply content/guides/api-calls/permissions-and-errors/common-errors.md:117]: Also how is this related to this PR?

File: content/guides/api-calls/permissions-and-errors/scopes.md [hunk 2]
```diff
@@ -201,9 +201,16 @@ Governance.
...
 
-This scope can be requested by opening a ticket via our support team, who will
-review request on an individual basis and only provide approval of the use case
-requires the scope.
+These scopes can be requested by opening a ticket via our
+[support team](page://support), who will review requests on an individual
+basis and only provide approval if the use case requires the scope.
+
+<Message type='notice'>
+  Scopes which are only available on request are not available for activation
```
Comment (cbetta) [inline content/guides/api-calls/permissions-and-errors/scopes.md:209]: Sentence reads a bit strange. Maybe change to: `It is not possible to request extra scopes if your account is a free trial account`?

File: content/guides/api-calls/permissions-and-errors/common-errors.md (hunk 1 above)
Comment (cbetta) [reply content/guides/api-calls/permissions-and-errors/common-errors.md:117]: Ignore me, seen the PR comment

File: content/guides/api-calls/permissions-and-errors/scopes.md (hunk 2 above)
Comment (jcleblanc) [reply content/guides/api-calls/permissions-and-errors/scopes.md]: wfm, changes made

**Comment by jcleblanc** [comment]
@cbetta wording changes updated, should be good now.

**Pull request merged**